### PR TITLE
Feat get skills

### PR DESCRIPTION
**Pull request by OKThomas1**
I still need to do proper documentation, but main features are:
Different admin styling, uses bootstrap and has a nav bar similar to the main site
A get skills page that allows admins to scrape jobs by job title and location, and then extract skills from the scraped jobs
A review skills page that allows admins to sort skills as valid, invalid, or invalid for specific job

### CODE REVIEW

File: skills_DS/skills_DS/settings.py [hunk 1]
```diff
@@ -87,7 +87,7 @@
         'NAME': 'skills_analysis',
         'USER': 'postgres',
         'PASSWORD': 'example',
-        'HOST': 'postgres',
+        'HOST': '127.0.0.1',
```
Comment (brianzhouzc) [inline skills_DS/skills_DS/settings.py]: The host needs to be `postgres` for it to work in Docker. Let me know if you need help setting up the Docker enviorment.

File: .gitignore [hunk 2]
```diff
@@ -2,4 +2,5 @@ env
 venv
 skills_DS/venv
 __pycache__/
-docker/postgresql/data
\ No newline at end of file
+secrets.json
```
Comment (brianzhouzc) [inline .gitignore:5]: Nice

File: skills_DS/api/skills_extraction.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
+from .models import Location, JobPosting, JobTitle, Skill
+import re
+import math
+import nltk
+
+nltk.download('stopwords')
+nltk.download('punkt')
+nltk.download('averaged_perceptron_tagger')
+nltk.download('wordnet')
+nltk.download('words')
```
Comment (brianzhouzc) [inline skills_DS/api/skills_extraction.py:10]: These should be added in the docker setup script instead.

File: skills_DS/api/views.py [hunk 4]
```diff
@@ -20,7 +26,74 @@ def post(self, request, format=None):
...
+					query.verified = True
+					query.save()
+				if skill['value'] == "invalid":
+					InvalidSkill.objects.get_or_create(job_title=job_title, name=skill['skill'], specific=False)
+					query.delete()
+				if skill['value'] == "invalid2":
+					InvalidSkill.objects.get_or_create(job_title=job_title, name=skill['skill'], specific=True)
+					query.delete()
+			else:
+				print("could not get skill")
```
Comment (brianzhouzc) [inline skills_DS/api/views.py]: This should return an error response.

File: skills_DS/api/skills_extraction.py (hunk 3 above)
Comment (brianzhouzc) [reply skills_DS/api/skills_extraction.py:10]: Don't worry about it for now - I will move them to the setup script once I merge the pdf parsing branch.

**Review by brianzhouzc — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 2 above)
Comment (brianzhouzc) [reply .gitignore:5]: We should have a `secrets_example.json` file with dummy data for reference.

File: skills_DS/skills_DS/settings.py (hunk 1 above)
Comment (OKThomas1) [reply skills_DS/skills_DS/settings.py]: I thought I stashed that change, but I probably added it back after I merged with develop. Next time I'll be sure to double check that I don't push that change.

**Review by SanaBob — APPROVED** [review]

**Review by brianzhouzc — APPROVED** [review]

**Pull request merged**